test/framework/cluster_proxy.go

```diff
@@ -253,6 +259,27 @@ func (p *clusterProxy) Apply(ctx context.Context, resources []byte, args ...stri
 	return exec.KubectlApply(ctx, p.kubeconfigPath, resources, args...)
 }
 
+// Create creates using the controller-runtime client.
+func (p *clusterProxy) Create(ctx context.Context, resources []byte) error {
+	Expect(ctx).NotTo(BeNil(), "ctx is required for Create")
+	Expect(resources).NotTo(BeNil(), "resources is required for Create")
+
+	var retErrs []error
+	objs, err := yaml.ToUnstructured(resources)
+	if err != nil {
+		return err
+	}
+	for o := range objs {
+		if err := p.GetClient().Create(ctx, &objs[o]); err != nil {
+			if apierrors.IsAlreadyExists(err) {
+				continue
+			}
```

Comment:
`kubectl create` fails when the resource already exists, while `apply` informs (through logging) that the resource already exists, and updates it in place.

What problem are we trying to solve here?

Reply:
See: https://github.com/kubernetes-sigs/cluster-api/pull/9736#issuecomment-1819363283